src/twisted/logger/_observer.py

```diff
@@ -136,17 +91,23 @@ def trace(observer):
                 observer=brokenObserver,
             )
 
-    def _errorLoggerForObserver(self, observer):
+    def _errorLoggerForObserver(self, observer: ILogObserver) -> Logger:
         """
         Create an error-logger based on this logger, which does not contain the
         given bad observer.
 
         @param observer: The observer which previously had an error.
-        @type observer: L{ILogObserver}
 
-        @return: L{None}
+        @return: A L{Logger} without the given observer.
         """
         errorPublisher = LogPublisher(
             *[obs for obs in self._observers if obs is not observer]
         )
         return Logger(observer=errorPublisher)
+
+
+@implementer(ILogObserver)
+def bitbucketLogObserver(event: LogEvent) -> None:
```

Comment:
Used where `lambda e: None` was used, but (a) it's a single object and (b) it's property typed.